capgains/cli.py

```diff
@@ -16,8 +16,9 @@ def capgains():
 @click.argument('transactions-csv')
 @click.option('-t', '--tickers', metavar='TICKERS',
               multiple=True, help="Stocks tickers to filter for")
-def show(transactions_csv, tickers):
-    transactions = TransactionsReader.get_transactions(transactions_csv)
+@click.option('--reverse/--no-reverse', default=False)
```

Comment:
I think it would be useful if you could add a small help string about what this option does.

Also is the `--no-reverse` option needed? A user can just avoid specifying the `--reverse` option to get the default chronological order for the table.